colcon-build-and-test/action.yaml

```
@@ -51,21 +51,36 @@ runs:
         . /opt/ros/${{ inputs.rosdistro }}/setup.sh
         colcon build --event-handlers console_cohesion+ \
           --packages-above-and-dependencies ${{ inputs.target-packages }} \
-          --cmake-args -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+          --cmake-args \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_C_FLAGS="--coverage -DCOVERAGE_RUN=1" \
+            -DCMAKE_CXX_FLAGS="--coverage -DCOVERAGE_RUN=1" \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+      shell: bash
+
+    - name: Install python package for coverage commands
+      run: |
+        pip3 install colcon-lcov-result
+        pip3 install colcon-coveragepy-result
```

How about this? Or you can add `\` and a new line.

```suggestion
    - name: Install Python packages for coverage commands
      run: |
        pip3 install colcon-lcov-result colcon-coveragepy-result
```